assets/js/theme/custom/mega-menu.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Open mega menu on search button click
+ */
+export default function megaMenu() {
+    const MOBILE_WIDTH = 768
+    const megaMenuElement = document.querySelector('.js-navigation-megamenu')
+    const searchButton = document.querySelector('.js-mega-menu-search')
+    const searchInputs = document.querySelectorAll('.js-search-quick')
+    const showMore = document.querySelectorAll('.js-show-more')
+    let hasAttachedEvent = false
+
+    const openDesktopMenu = () => {
+        megaMenuElement.previousElementSibling.classList.toggle('is-open')
```

Comment:
This could be accidentally broken by basic html edit. Please add a dedicated js classname for the element instead of using `previousElementSibling`